doc/whats_new.rst

```diff
@@ -36,6 +36,8 @@ Changelog
 
     - Add support for Brainvision v2 by `Teon Brooks`_
 
+    - Animation for evoked topomap by `Jaakko Leppakangas`_
```

Comment:
it's still a bit broken on my mac. it cannot be a priority for me right
now. I propose to postpone this after the release in january.